src/asset/index.ts

```diff
@@ -3,22 +3,26 @@ import {getAddress} from "../rpc/getAddress";
 import {executeAssetOperation} from "./action";
 import formatBalance from "@polkadot/util/format/formatBalance";
 import {Balance} from "@polkadot/types/interfaces";
+import {NetworkConfiguration} from "../network/interfaces";
 
 const assets: Map<string, Asset> = new Map<string, Asset>();
 
-
 function getIdentifier(origin: string, id: string): string {
   return `${origin}_${id}`;
 }
 
-export function getKusamaAsset(assetId: string, balance: number|string|Balance, address: string): Asset {
+// eslint-disable-next-line max-len
+export function getKusamaAsset(assetId: string, balance: number|string|Balance, address: string, configuration: NetworkConfiguration): Asset {
+  const image = configuration.unit.image ? configuration.unit.image : "";
+  const customViewUrl = configuration.unit.customViewUrl ?
+    configuration.unit.customViewUrl : `https://polkascan.io/pre/kusama/account/${address}`;
```

Comment:
I would suggest that you use deepmerge lib to merge default config into provided one so you can be sure here that all params are set

Reply:
you can do it mannually if you don't wish to use another lib. It's small enough that it can be done mannually